### PR TITLE
Adds a craftable baseball bat

### PR DESCRIPTION
**Pull request by lzimann**
:cl: Lzimann & Keekenox
add: You can now craft a baseball bat with 5 sheets of wood!
/:cl:

The baseball bat has 10 force now, althought it throws the person back one or two tiles.

fix token: #19090 

Sprites by Keekenox

In-hand sprites:

inhand

Item sprite:

bat

### CODE REVIEW

**Comment by Cheridan** [comment]
stupidly strong for something that's incredibly easy to obtain

**Comment by swankcookie** [comment]
can we add baseballs at least so that this has some semblance of a purpose?

**Comment by Aloraydrel** [comment]
Why esword when you can baseball!

**Comment by coiax** [comment]
Why this isn't hand crafted requiring at least some sort of cutting tool or knife, I dunno

**Comment by WJohn** [comment]
This should probably be 15 brute, as those are fairly uncommon already.

**Comment by Zilenan91** [comment]
15 brute sounds nice for this, 20 is a bit too high. Otherwise :+1:

**Comment by jughu** [comment]
make baseball bat only do 20 when it hits the head and if it hits your legs you are crippled 4 life

**Comment by Fox-McCloud** [comment]
15 seems reasonable for this---if it was a two-handed weapon, then I'd say 18 would be reasonable, but a single handled 20 force weapon is ...ridiculous.

**Comment by lzimann** [comment]
I'm still thinking about how I'm going. Cheridan suggested something really cool and I'm strongly leaning towards that.

**Comment by Zilenan91** [comment]
What'd he suggest

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -298,3 +298,13 @@
 /obj/item/weapon/melee/skateboard/attack_self(mob/user)
 	new /obj/vehicle/scooter/skateboard(get_turf(user))
 	qdel(src)
+
+/obj/item/weapon/melee/baseball_bat
+	name = "baseball bat"
+	desc = "There ain't a scull in the league that can withstand a swatter"
```
Comment (Jordie0608) [inline code/game/objects/items/weapons/weaponry.dm]: scull

Comment (lzimann) [inline code/game/objects/items/weapons/weaponry.dm]: i just copied from someone, not my fault i swear!

**Comment by KingDairy** [comment]
traitor batter outfit that lets you deflect projectiles back at whoever shot/threw them when
also something about ghosts

File: code/game/objects/items/weapons/weaponry.dm [hunk 2]
```diff
@@ -302,9 +302,15 @@
 /obj/item/weapon/melee/baseball_bat
 	name = "baseball bat"
 	desc = "There ain't a scull in the league that can withstand a swatter"
```
Comment (coiax) [inline code/game/objects/items/weapons/weaponry.dm]: .

**Comment by kilog** [comment]
@KingDairy hecks yes do it tbh

**Comment by lzimann** [comment]
@KingDairy i might code it after this gets merged

**Comment by KorPhaeron** [comment]
Make it hit block thrown items/hit them back at people

**Pull request merged**